docs/local.md

```diff
@@ -0,0 +1,26 @@
+# Local Development
+
+A local development environment is provided by a [Docker Compose](https://docs.docker.com/compose/) configuration.
+
+
+## First-time setup
+
+A few steps are required the first time
+
+1. [Install Docker Engine](https://docs.docker.com/engine/installation/).
```

Comment:
Maybe point to Docker Toolbox instead? The download page for mac says, "Note: This release of Docker deprecates the Boot2Docker command line in favor of Docker Machine. Use the Docker Toolbox to install Docker Machine as well as the other Docker tools."